canonical-web-and-design/maas-ui
# Break down KVM list into header, table and cell components.

## Done
- Broke down KVMList component into logical sub-components:
  - KVMListHeader, which will include the action forms eventually,
  - KVMListTable, which is basically a prop'd MainTable,
    - Each cell is also split into its own component, like with the machine list.
- Power and OS cells just return "Unknown" for now. They will be handled as a separate issue because they require a bit of extra work to find the relevant host info.
- Adjusted the meter label padding so it matches default line break spacing.

## QA
- Go to /MAAS/r/kvm.
- Check that the cell data matches the data in /MAAS/l/kvm, except for Power and OS.

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1951
